sound/soc/sof/intel/hda-stream.c

```diff
@@ -93,9 +94,6 @@ static int hda_setup_bdle(struct snd_sof_dev *sdev,
 		bdl++;
 		hstream->frags++;
 		offset += chunk;
-
-		dev_vdbg(sdev->dev, "bdl, frags:%d, chunk size:0x%x;\n",
-			 hstream->frags, chunk);
```

Comment:
maybe we should remove this in a separate patch added earlier to avoid questions as to why we removed this without a tracepoint added?